distribution/lib/Standard/Database/0.0.0-dev/src/Data/DB_Table.enso

```diff
@@ -2786,20 +2786,20 @@ display_dataframe df indices_count all_rows_count format_terminal =
 ## PRIVATE
    By default, join on the first column, unless it's a cross join, in which
    case there are no join conditions.
-default_join_condition : Table -> Join_Kind | Join_Kind_Cross -> Join_Condition
+default_join_condition : DB_Table -> Join_Kind | Join_Kind_Cross -> Join_Condition
 default_join_condition table join_kind = case join_kind of
     Join_Kind_Cross.Cross -> []
     _ -> [Join_Condition.Equals table.column_names.first]
 
 ## PRIVATE
-Materialized_Table.from (that:Table) =
+Materialized_Table.from (that:DB_Table) =
```

Comment:
While I like the explicitness of `Materialized_Table`, I'm not exactly sure if the renaming is still needed if the names are no longer clashing.

This could just be `Table.from (that : DB_Table)`.

But I'm not attached to this suggestion - I'm a bit on the fence between the two approaches, so just wanted to share, but whichever you think makes more sense.

Reply:
Next MR removes aliases. But I think now the name clash has gone we don't need them.